Documentation/docs/index.md

```diff
@@ -30,6 +30,7 @@ download
 Examples <https://examples.itk.org>
 Tutorial <https://github.com/InsightSoftwareConsortium/ScientificImageAnalysisVisualizationAndArtificialIntelligenceCourse>
 Software Guide <https://itk.org/ItkSoftwareGuide.pdf>
+learn/faq.md
```

Comment:
We will want to change the `:maxdepth:` above to `1` to address:

![image](https://github.com/InsightSoftwareConsortium/ITK/assets/25432/4c41e4ac-488c-4b00-a10b-3e7b91e3267b)

Reply:
:+1: Done in e808466.